lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -0,0 +1,69 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class JavascriptBigInt sealed : public RecyclableObject
+    {
+
+    private:
+        Field(BOOL) m_isNegative;
```

Comment:
nit: `bool` is preferred over `BOOL`

Reply:
any reason about this preference?

Reply:
the all-upper type names are generally windows-specific and we have simply shimmed them out on Linux. More importantly in bool vs BOOL, BOOL as defined by the Windows SDK is 32bits (it came from before there was a standard c/c++ bool type, so it was just typedef'd to int). Also, I just think the lowercase names look better, but that's minor :)